pkg/codegen/testing/test/testdata/naming-collisions/python/pulumi_example/mod/component.py

```diff
@@ -8,8 +8,8 @@
 import pulumi.runtime
 from typing import Any, Mapping, Optional, Sequence, Union, overload
 from .. import _utilities
-from ..component2 import Component2
 from ..main_component import MainComponent
+from .component2 import Component2
```

Comment:
I am fairly certain that the current snapshot here was wrong.

Reply:
This is why we need conformance testing, snapshots don't actually check things are correct.